xcode/Subconscious/Shared/Components/Notebook/Notebook.swift

```diff
@@ -152,6 +152,7 @@ enum NotebookAction {
     /// request an editor detail.
     case findAndPushDetail(
         slashlink: Slashlink,
+        traversalPath: TraversalPath,
```

Comment:
If this is part of the location, would prefer to find a way to model this in Slashlink or MemoAddress

Reply:
Dropped `TraversalPath` and replaced with `MemoAddress` everywhere.